gistUploader.js

```diff
@@ -1,11 +1,20 @@
-
 const axios = require('axios');
+const fs = require('fs');
+
+const API_URL = 'https://api.github.com/gists';
+const fileName = process.argv[3];
+const post = {};
+
+function processFile(fileContents) {
+  post[fileName] = { content: fileContents };
```

Comment:
don't access variables outside of the scope. Here you can receive `fileName` as a function's parameter

Reply:
done